_data/authors.yml

```diff
@@ -74,3 +74,9 @@ ula:
   email : ula.stankiewicz@brightinventions.pl
   bio : "Employer Branding Specialist @ Bright Inventions"
   avatar : http://brightinventions.pl/images/ula.jpg
+
+agnieszka:
+  name : Agnieszka Olszewska
+  email : agnieszka.olszewska@brightinventions.pl
+  bio : "Software Engineer @ Bright Inventions"
+  avatar : http://brightinventions.pl/images/agnieszka.jpg
```

Comment:
Should probably be `http://brightinventions.pl/images/agnieszka.png`.

Reply:
You're right. Corrected.